src/frontend/src/components/groups/GroupRow/GroupRow.tsx

```diff
@@ -0,0 +1,117 @@
+import { Avatar, Box, Card, CheckBox, Grid, Text } from 'grommet';
+import React, { ChangeEvent, useRef, useState } from 'react';
+import { More } from 'grommet-icons';
+import { useMediaQuery } from 'react-responsive';
+
+export interface Group {
+  id: string;
+  name: string;
+  members: {
+    id: string;
+    name: string;
+    avatar: string;
+  }[];
+}
+
+export interface GroupRowProps {
+  /**
+   * The group to display
+   */
+  group: Group;
+  /**
+   * Callback when the group is selected or deselected
+   */
+  onToogle: (selected: boolean) => void;
+  /**
+   * Is the group currently selected
+   */
+  selected: boolean;
+}
+
+export default function GroupRow(props: GroupRowProps) {
+  const { group, onToogle, selected } = props;
```

Comment:
```suggestion
```

Reply:
I'm not used to destructure props directly, but we can commit to that indeed.